### ISSUE TITLE
Bug - Firefox strips leading spaces from strings

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#4619]
## Description
Text within the `hidden` tag cannot begin with a space and text adjacent to the closing `hidden` tag cannot begin with a space as it is stripped away in Firefox and alters the copy for screen readers.

## Screenshot
![Screen Shot 2022-11-07 at 09 58 42](https://user-images.githubusercontent.com/3046459/200344704-ea260ea5-7001-449a-a661-fd3d989dda90.png)

## Acceptance Criteria
- [ ] update strings so that hidden strings are read properly by screen readers
- [ ] ensure French translations do not get disconnected and also are update appropriately

**Comment by mnigh** [comment]
See: https://github.com/GCTC-NTGC/gc-digital-talent/pull/4837#issuecomment-1322709079
